### PR TITLE
cpp-netlib 0.11.1RC2 (new formula)

### PR DESCRIPTION
**Pull request by alexnauda**
Modified from seemingly abandoned PR https://github.com/Homebrew/homebrew/pull/29511
- upgraded to 0.11.1RC2
- added a test

### CODE REVIEW

File: Library/Formula/cpp-netlib.rb [hunk 1]
```diff
@@ -0,0 +1,42 @@
+require "formula"
+
+class CppNetlib < Formula
+  homepage "http://cpp-netlib.org"
+  url "http://storage.googleapis.com/cpp-netlib-downloads/0.11.1/cpp-netlib-0.11.1RC2.tar.gz"
```
Comment (guoxiao) [inline Library/Formula/cpp-netlib.rb]: The version 0.11.1 final is out.  http://cpp-netlib.org/

Comment (guoxiao) [inline Library/Formula/cpp-netlib.rb]: see #35804

**Comment by adamv** [comment]
This needs to be updated to the final version for further review.

File: Library/Formula/cpp-netlib.rb (hunk 1 above)
Comment (vszakats) [inline Library/Formula/cpp-netlib.rb]: My suggestion to switch to `https` from `http`, `storage.googleapis.com` supports it.

**Comment by tdsmith** [comment]
We pulled #35804 instead.

**Pull request closed without merging**